winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,65 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', '$state', '$stateParams', 'claimService', 'formTemplateService',
+  function($scope, $state, $stateParams, claimService, formTemplateService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+
+    $scope.allForms = formTemplateService; // returns an object {formId -> {...}}
+    var userForms = [ // TODO: state of all forms edited by user
```

Comment:
I'm not sure what you need this hardcoded object for, it seems like you just use it to iterate the possible forms that could be completed. You could just iterate through the keys in `formTemplateService` to the same effect, and that will avoid the hardcoding